deployments/deploy-ssh.sh

```diff
@@ -189,6 +198,8 @@ spec:
             "--app-list", "$APP_LIST",
             "--node-driver", "ssh",
             "--scale-factor", "$SCALE_FACTOR",
+            "--config-map", "$CONFIG_MAP",
```

Comment:
what is the use of -config-map in this PR?

Reply:
it's not needed in this PR, will delete.

Reply:
This is still present.

Reply:
Deleted.